lib/Common.js

```diff
@@ -196,7 +196,7 @@ Common.prepareAppConf = function(opts, app) {
     } else if (f != 'log' || (f == 'log' && af)) {
       ps = [cwd, af];
 
-      if (!fs.existsSync(path.dirname(af))) {
+      if (!fs.existsSync(path.dirname(path.resolve(cwd, af)))) {
         Common.printError(cst.PREFIX_MSG_WARNING + 'Folder does not exists: ' + path.dirname(af));
         Common.printOut(cst.PREFIX_MSG + 'Creating folder: ' + path.dirname(af));
         mkdirp(path.dirname(af), function(err) {
```

Comment:
i think you should make the same here : `mkdirp(path.dirname(path.resolve(cwd, af))` ?